decidim-templates/app/packs/src/decidim/templates/admin/block_template_chooser.js

```diff
@@ -0,0 +1,10 @@
+$(() => {
+  $("#block_template_chooser").change(function() {
+    let dropDown =  $("#block_template_chooser");
+    $.getJSON(dropDown.data("url"), {
+      id: dropDown.val()
+    }).done(function(data) {
+      $("#block_user_justification").val(data.template);
+    });
+  });
+});
```

Comment:
We're planning to drop jQuery in the future, and with that in mind we're trying to not introduce code dependent on it. This can be a good starting point, feel free to modify it: 

```suggestion
// Choose a Block User Message template, get it by AJAX and add the Template in the justification textarea
document.addEventListener("DOMContentLoaded", () => {
  document.getElementById("block_template_chooser").addEventListener("change", () => {
    const dropdown =  document.getElementById("block_template_chooser");
    const url = dropdown.getAttribute("data-url");
    const templateId = dropdown.value; 

    if (templateId === "") {
      return;
    }

    fetch(`${new URL(url).pathname}?${new URLSearchParams({ id: templateId })}`).
      then((response) => response.json()).
      then((data) => {
        document.getElementById("block_user_justification").value = data.template;
      })
  });
});
```